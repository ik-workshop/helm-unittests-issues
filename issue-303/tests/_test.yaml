---
suite: issue 303 for helm-unittest
tests:
- it: manifest should match snapshot
  asserts:
    - matchSnapshot: {}
# - it: should have the service first
#   documentIndex: 0
#   asserts:
#   - isKind:
#       of: Service
  # - equal:
  #     path: metadata.name
  #     value: example-service

# - it: should have the deployment second
#   documentIndex: 1
#   asserts:
#   - isKind:
#       of: Deployment
  # - equal:
  #     path: metadata.name
  #     value: example-deployment

- it: manifest should match snapshot
  asserts:
    - matchSnapshot: {}
