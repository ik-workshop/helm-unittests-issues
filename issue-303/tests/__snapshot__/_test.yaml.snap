manifest should match snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      name: example-deployment
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      name: example-service
